docker-compose.yml

```diff
@@ -67,7 +67,8 @@ services:
     volumes:
       - ./:/src:z
       - ./docker/message_broker/certs:/broker-certs:ro,z
-      - request-logs-volume:/var/log/iib/requests:z
+      - request-logs-volume:/var/log/iib/requests/request_logs:z
+      - request-related-bundles-volume:/var/log/iib/requests/related_bundles:z
```

Comment:
Maybe the related bundles information shouldn't go under `/var/log`? Is `/var/lib` a better path?